src/py/pyodide/webloop.py

```diff
@@ -167,6 +167,10 @@ def wrapper(fut: Future[T]) -> None:
         return result
 
 
+class PyodideTask(Task[T], PyodideFuture[T]):
```

Comment:
I'm wondering if we should have called it just `Future` and  `Task` so you could do,
```
from pyodide.webloop import Future, Task
```
as a drop-in replacement of whatever asyncio user had. Probably too late now anyway (and I'm not 100 % that was a good idea in the first place).

Reply:
In the docstring it would be good to explain when would one want to use this instead of `asyncio.Task`  also shouldn't we expose this in the docs as well?

Reply:
The user isn't really supposed to use it directly. The documentation for `Task` says:

> Manual instantiation of Tasks is discouraged.

Yeah it's a good idea to add a bit of documentation.